types/react-d3-graph/index.d.ts

```diff
@@ -94,6 +94,7 @@ export interface GraphConfiguration<N extends GraphNode, L extends GraphLink> {
     highlightOpacity: number;
     maxZoom: number;
     minZoom: number;
+    initialZoom: number;
```

Comment:
is that property optional or it accepts a null?
```txt
initialZoom (number = null) 🔗 initial zoom that can be set on the graph.
```
https://github.com/danielcaldas/react-d3-graph/blob/e5cd27ecacdfc4369f9b76f7ad9416d733ac10a9/src/components/graph/graph.config.js#L90

The property seems to be optional. 
If you include a test, you will be able to self merge your change. thx!